modules/infra/submodules/storage/main.tf

```diff
@@ -24,6 +24,18 @@ locals {
       policy_json = data.aws_iam_policy_document.costs[0].json
       arn         = aws_s3_bucket.costs[0].arn
     } : {}
+    flyte_metadata = var.storage.flyte.enabled ? {
```

Comment:
`var.flyte.enabled` ?

Reply:
Yeah sorry was running into issues on the platform apps side so I tried adding `storage` here as a sanity check. It didn't resolve things, so undoing this commit